lib/nucore.rb

```diff
@@ -108,6 +108,10 @@ def self.included(base)
       end
       module ClassMethods
 
+        def order_by_asc_nulls_first(field)
+          NUCore::Database.oracle? ? order("#{field} asc nulls first") : order(field)
```

Comment:
CodeClimate is warning about SQL injection on this line. Is there some reason the `?` format won't work here? Otherwise, can we update this line and the corresponding line below?